pulsar-broker/src/main/java/com/yahoo/pulsar/broker/PulsarService.java

```diff
@@ -95,6 +95,8 @@
     private LocalZooKeeperConnectionService localZooKeeperConnectionProvider;
     private final ScheduledExecutorService executor = Executors.newScheduledThreadPool(20,
             new DefaultThreadFactory("pulsar"));
+    private final ScheduledExecutorService cacheExecutor = Executors.newScheduledThreadPool(10,
```

Comment:
we may configure number of netty-io and executor threads but I think we can create an issue and should address into different PR.